src/Microsoft.Azure.WebJobs.ServiceBus/EventHubs/EventHubTriggerAttribute.cs

```diff
@@ -35,6 +36,7 @@ public EventHubTriggerAttribute(string eventHubName)
         /// <summary>
         /// Optional connection name. If missing, tries to use a registered event hub receiver.
         /// </summary>
+        [AppSetting]
         public string Connection { get; set; }
```

Comment:
It's automatically resolved when the attribute is cloned and passed to BindToInput rule